rdpwrap.ini

```diff
@@ -9050,11 +9050,14 @@ LocalOnlyPatch.x64=1
 LocalOnlyOffset.x64=8AF21
 LocalOnlyCode.x64=jmpshort
 SingleUserPatch.x64=1
-SingleUserOffset.x64=718EC
+;SingleUserOffset.x64=718EC
+SingleUserOffset.x64=978DB

```

Comment:
Strange value... Why? @loyejaotdiqr47123 
Any logic?

Reply:
This is really strange, you can look at [Pull Request #333 - sebaxakerhtc/rdpwrap.ini](https://github.com/sebaxakerhtc/rdpwrap.ini/pull/333),the wrong offset works fine.